gui/default/syncthing/core/syncthingController.js

```diff
@@ -2,7 +2,7 @@ angular.module('syncthing.core')
     .config(function ($locationProvider) {
         $locationProvider.html5Mode({ enabled: true, requireBase: false }).hashPrefix('!');
     })
-    .controller('SyncthingController', function ($scope, $http, $location, LocaleService, Events, $filter, $q, $compile, $timeout, $rootScope, $translate) {
+    .controller('SyncthingController', function ($scope, $http, $location, LocaleService, Events, $filter, $q, $compile, $timeout, $rootScope, $translate, percentFilter, binaryFilter) {
```

Comment:
Where do these two come from actually?  I can't find any change where these arguments are added to a function call?

Reply:
These are separate files:

https://github.com/syncthing/syncthing/blob/main/gui/default/syncthing/core/binaryFilter.js
https://github.com/syncthing/syncthing/blob/main/gui/default/syncthing/core/percentFilter.js

Reply:
I just don't get how adding an argument to a function *definition* makes something show up somewhere when the place where the function is called doesn't get any adjustment.

But on the other hand, I have no idea whatsoever how this whole AngularJS magic works, so if it works in testing (which I haven't seen because no folders had that status), then fine with me.

Reply:
Yeah, they do work fine in my testing. I've managed to find this out by analysing how https://github.com/syncthing/syncthing/blob/main/gui/default/syncthing/core/localeService.js is used in a similar manner, and also thanks to https://stackoverflow.com/questions/14302267/how-to-use-a-filter-in-a-controller.